Cargo.toml

```diff
@@ -124,6 +124,7 @@ doc-comment = "0.3"
 tokio = { version = "0.2.0", default-features = false, features = ["macros"] }
 
 [target.'cfg(windows)'.dependencies]
+regex = "1"
```

Comment:
Regex can be a pretty big dependency. Is it feasible to implement this without regular expressions?

Reply:
Sure, the latest commit performs the same check without using regex.